diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/UnsafeUseLastIndex.kt

```diff
@@ -0,0 +1,52 @@
+package org.cqfn.diktat.ruleset.rules.chapter6
+
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.DiktatRule
+import org.cqfn.diktat.ruleset.utils.*
+
+import com.pinterest.ktlint.core.ast.ElementType.BINARY_EXPRESSION
+import com.pinterest.ktlint.core.ast.ElementType.DOT_QUALIFIED_EXPRESSION
+import com.pinterest.ktlint.core.ast.ElementType.INTEGER_CONSTANT
+import com.pinterest.ktlint.core.ast.ElementType.OPERATION_REFERENCE
+import com.pinterest.ktlint.core.ast.ElementType.REFERENCE_EXPRESSION
+import com.pinterest.ktlint.core.ast.children
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+/**
+ * This rule checks if there use property length with operation - 1 and fix this on lastIndex
+ */
+class UnsafeUseLastIndex(configRules: List<RulesConfig>) : DiktatRule(
+    "last-index",
+    configRules,
+    listOf(Warnings.UNSAFE_USE_LAST_INDEX)
```

Comment:
`UNSAFE_USE_LAST_INDEX` please rename it to something more accurate: `USE_LAST_INDEX`

Reply:
Done